tools/docker/docker-compose-dev.yaml

```diff
@@ -17,6 +17,7 @@ x-environment:
   - EDA_ANSIBLE_RULEBOOK_LOG_LEVEL=${EDA_ANSIBLE_RULEBOOK_LOG_LEVEL:-'-v'}
   - EDA_CONTAINER_NAME_PREFIX=${EDA_CONTAINER_NAME_PREFIX:-eda}
   - EDA_PODMAN_ENV_VARS=${EDA_PODMAN_ENV_VARS:-'@none None'}
+  - EDA_PGSSLMODE=${EDA_PGSSLMODE:-disable}
```

Comment:
If the default value is "allow" we don't need to modify anything else and docker-compose would work as it is, don't?

Reply:
@Alex-Izquierdo yep, I believe I forgot to remove this when squashing the commits.